crowdin/messages.json

```diff
@@ -19,6 +19,7 @@
   "24062713": "0.2 },",
```

Comment:
Please update your PR with the base branch ([niloo/89003/refactoring-translations-parent](https://github.com/niloofar-deriv/deriv-com/tree/niloo/89003/refactoring-translations-parent))
The reason for that is we updated the key extractions configs https://github.com/binary-com/deriv-com/pull/4117, since all of the merges to the project is blocked right now I added the same configs to this branch so by having this your crowdin wouldn't update like this.